lib/generators/stimulus_reflex/templates/app/javascript/config/stimulus_reflex.js.tt

```diff
@@ -0,0 +1,5 @@
+import { application } from "../controllers/application"
+import controller from '../controllers/application_controller'
+import StimulusReflex from 'stimulus_reflex'
+
+window.reflexes = StimulusReflex.initialize(application, { controller, isolate: true })
```

Comment:
I still think this is confusing and redundant if you can access the same thing via `StimulusReflex.reflexes`
```suggestion
StimulusReflex.initialize(application, { controller, isolate: true })
```

Reply:
I still disagree... and I'm going to go with your perspective, because this is a collective effort. 💙